src/Arrays/MPIStateArrays.jl

```diff
@@ -4,31 +4,28 @@ using DoubleFloats
 using LazyArrays
 using StaticArrays
 using GPUifyLoops
+import ..haspkg
```

Comment:
```suggestion
using ..haspkg
```

Reply:
Doesn't work for me locally to do it this way.

Reply:
That's odd, why is that? @simonbyrne